hub/api/tests/test_api.py

```diff
@@ -694,6 +695,7 @@ def test_htypes_list():
         "image",
         "class_label",
         "bbox",
+        "audio",
```

Comment:
is audio different from mp3?

Reply:
Audio is htype, mp3 is compression. We can add other audio compressions like wav in the future.